students/charlie_england/session03/list_lab.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+print("working")
+
+#Series 1
+lst = ["Apples", "Pears", "Oranges","Peaches"]
+print(lst)
+lst.append(input("Please add another fruit"))
+print(lst)
+user_input = input("Please enter a number >>>")
+#print(f"{user_input} cooresponds to {lst[int(user_input)-1]}")
```

Comment:
You may want to uncomment this :)

Reply:
Actually I may ask for help on this in after class. I commented it out because visual studio code kept throwing an error. When I would run it in VS code, 2 things would be "auto-inputted" 'C:/Users/Charl/Anaconda3/Scripts/activate'] and conda activate base so if I have an input that requires a number I get an error thrown. I'll go ahead and uncomment this since I'm fixing the mailroom.py code.